pymeasure/display/curves.py

```diff
@@ -75,6 +76,75 @@ def update(self):
 
 # TODO: Add method for changing x and y
 
+#############################################################################################
+
+class ResultsImage(pg.ImageItem):
+    """ Creates an image loaded dynamically from a file through the Results
+    object."""
+    def __init__(self, results, x, y, z, force_reload=False):
+
+        self.results = results
+        self.x = x
+        self.y = y
+        self.z = z
+        # NOTE: Below we're making an assumption on the form of *any* procedure which we would use
+        # with an image. Seems reasonable, we need to make some compromise so that
+        # we know how to construct the resolution and so that we know what the heck the range is
+        self.xstart = getattr(self.results.procedure,self.x + '_start')
```

Comment:
The `getattr` calls should have a space between the arguments to be PEP8.

Reply:
Thanks, it's been fixed. There are probably more PEP8 violations in what I've written, so let me know if you find any more.